content/blog/2018/custom-gateway/index.md

```diff
@@ -0,0 +1,362 @@
+---
+title: Add a new Custom IngressGateway
+description: Describes how to add a new custom ingress gateway manually.
+publishdate: 2018-10-26
+subtitle: Custom Ingress Gateway
+attribution: Julien SENON
+weight: 89
+keywords: [ingress,traffic-management, aws]
+---
+
+This post provide instructions to create manually a new custom ingress gateway with automatic provisioning of certificate based on cert-manager.
+
+This post was validated on AWS Infrastructure
+
+## Prerequisites
+
+Have cert-manager installed with [chart](https://github.com/helm/charts/tree/master/stable/cert-manager)
+
+## Cert-Manager Configuration
+
+### Check installation
```

Comment:
Two headings without content in between indicate a problem with the content structure. In this case, all level 3 headings should be changed to level two and ## Cert-Manager configuration removed. If you do that, the text in the level 2 headings must also change: 
## Check the Cert-Manager installation

Reply:
Thx for your suggestion